src/shadowbox/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "outline-server",
+  "name": "shadowbox",
```

Comment:
I'd prefer to move away from the term "shadowbox". It's not a recognizable name. 'outline-server' is better in that sense. Why do you want to change?

Reply:
The top level package name is 'outline-server', which is consistent with the repository name. I think it makes sense to name the workspace differently.

Reverted to 'outline-server'.